polkadot/jemalloc-shim/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "polkadot-jemalloc-shim"
+version = "1.0.0"
+description = "Shim crate to enable jemalloc-allocator feature for polkadot crates"
+license.workspace = true
+authors.workspace = true
+edition.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+polkadot-cli = { optional = true, workspace = true }
+polkadot-node-core-pvf = { optional = true, workspace = true }
+polkadot-node-core-pvf-prepare-worker = { optional = true, workspace = true }
+polkadot-overseer = { optional = true, workspace = true }
+tikv-jemallocator = { optional = true, features = ["unprefixed_malloc_on_supported_platforms"], workspace = true }
```

Comment:
Maybe remove `unprefixed_malloc_on_supported_platforms` to avoid conflicting requirements (it seems that cumulus transitively requires jemalloc without this feature).

Reply:
The same feature is now enabled for omni-node and polkadot-parachains.